### ISSUE TITLE
Validation elements displaying in Chrome breaks selenoid

### ISSUE THREAD

**Issue by APshenkin** [aandryashin/selenoid#27]
Trying to validate elements displaying. To do this I complete `elements` request to get all elements that path selector and then for each element complete /session/:sessionId/element/:elementId/displayed' async request (this requests sends in the same time). When trying to complete 23 `displayed` selenoid breaks with this error
```
2017/03/17 13:36:51 Process 23529 started in: 57.337089ms
2017/03/17 13:36:51 Proxying requests to: http://127.0.0.1:59373
2017/03/17 13:36:51 [SESSION_ATTEMPTED] [http://127.0.0.1:59373]
2017/03/17 13:36:52 [SESSION_CREATED] [c763e80c0377b8e02fa9fc57c8fa56ca] [http://127.0.0.1:59373]
2017/03/17 13:37:00 http: proxy error: read tcp 127.0.0.1:59609->127.0.0.1:59373: read: connection reset by peer
2017/03/17 13:37:00 http: proxy error: dial tcp 127.0.0.1:59373: getsockopt: connection reset by peer
2017/03/17 13:37:00 http: proxy error: read tcp 127.0.0.1:59615->127.0.0.1:59373: read: connection reset by peer
2017/03/17 13:37:00 http: proxy error: read tcp 127.0.0.1:59617->127.0.0.1:59373: read: connection reset by peer
panic: close of closed channel

goroutine 241 [running]:
main.proxy.func1.3(0xc4204a2600)
	/Users/a.pshenkin/go/src/github.com/aandryashin/selenoid/selenoid.go:164 +0x1d6
net/http/httputil.(*ReverseProxy).ServeHTTP(0xc42047ff70, 0x145e640, 0xc4204981c0, 0xc42000be00)
	/usr/local/Cellar/go/1.8/libexec/src/net/http/httputil/reverseproxy.go:159 +0x1be
main.proxy.func1(0xc420489020, 0x145e640, 0xc4204981c0, 0xc42000be00)
	/Users/a.pshenkin/go/src/github.com/aandryashin/selenoid/selenoid.go:179 +0x117
created by main.proxy
	/Users/a.pshenkin/go/src/github.com/aandryashin/selenoid/selenoid.go:180 +0x7f
```

**Comment by vania-pooh** [comment]
@APshenkin wow, that's a bug. We'll fix soon.

**Comment by aandryashin** [comment]
It looks like chrome driver process crashed. Could you please try to execute the same request against chrome driver server?

**Comment by APshenkin** [comment]
@aandryashin When using chromedriver directly everything works great.

**Comment by vania-pooh** [comment]
@APshenkin any updates? Does it reproduce with latest version (```aandryashin/selenoid:latest```)?

**Comment by rovner** [comment]
Faced the same issue with iedriver
```
2017/04/05 06:05:41 [NEW_REQUEST]
2017/04/05 06:05:41 [NEW_REQUEST_ACCEPTED]
2017/04/05 06:05:41 Locating the service for internet explorer
2017/04/05 06:05:41 Using default version: 11
2017/04/05 06:05:41 Using driver service for internet explorer 11
2017/04/05 06:05:41 Trying to allocate port
2017/04/05 06:05:41 Available port is: 49980
2017/04/05 06:05:41 Starting process: [./headless_ie_selenium.exe --port=49980]
2017/04/05 06:05:43 Process 3816 started in: 2.1046319s
2017/04/05 06:05:43 Proxying requests to: http://127.0.0.1:49980
2017/04/05 06:05:43 [SESSION_ATTEMPTED] [http://127.0.0.1:49980]
2017/04/05 06:05:44 [SESSION_CREATED] [6fb6d690-7c0e-48bb-803e-a3dbd3659982] [http://127.0.0.1:49980]
panic: close of closed channel

goroutine 3410 [running]:
main.proxy.func1.3(0xc042036600)
        /src/testing/go/src/github.com/aandryashin/selenoid/selenoid.go:170 +0x1dd
net/http/httputil.(*ReverseProxy).ServeHTTP(0xc042367f70, 0x864140, 0xc0420d2000, 0xc042036400)
        /usr/local/Cellar/go/1.8/libexec/src/net/http/httputil/reverseproxy.go:159 +0x1c5
main.proxy.func1(0xc04203a300, 0x864140, 0xc0420d2000, 0xc042036400)
        /src/testing/go/src/github.com/aandryashin/selenoid/selenoid.go:185 +0x11e
created by main.proxy
        /src/testing/go/src/github.com/aandryashin/selenoid/selenoid.go:186 +0x86
```

**Comment by aandryashin** [comment]
Should be fixed in 9b0a7c1614c0701c4c692ecb2d7bfba3d29aec69 commit, please check if the issue still persists.

**Comment by vania-pooh** [comment]
Can't reproduce now. Closing. Feel free to reopen if the issue persists with latest container version.
